Remove the *.png file installation from the frr-doc package. It doesn't look
like it's installed there with Bullseye's tools.

From: Saikrishna Arcot <sarcot@microsoft.com>

---
 debian/frr-doc.install |    1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/frr-doc.install b/debian/frr-doc.install
index c48dc5a8d..955c8fc5a 100644
--- a/debian/frr-doc.install
+++ b/debian/frr-doc.install
@@ -3,7 +3,6 @@ usr/share/doc/frr/html
 
 # info + images referenced by it
 usr/share/info/
-doc/user/_build/texinfo/*.png usr/share/info
 
 # other
 README.md         usr/share/doc/frr
